addon/Feeds/TopSitesFeed.js

```diff
@@ -25,7 +26,9 @@ module.exports = class TopSitesFeed extends Feed {
       if (experiments.screenshots) {
         try {
           links = yield getScreenshots(links, site => {
-            if (site.favicon_height >= 64 && site.favicon_width >= 64) {
+            if (site.favicon_height >= 64 && site.favicon_width >= 64 && ["/", ""].includes(urlParse(site.url).pathname)) {
+              // If we are at the "root domain path" and the icon is big enough,
+              // we don't show a screenshot.
               return false;
             }
             return true;
```

Comment:
@rlr Can you add this change right here:

https://github.com/JaredKerim-Mozilla/activity-stream/commit/187f3588f96995992fde794825835be8b0ac01b3